docs/en/cloud/AWS-CFT.md

```diff
@@ -0,0 +1,261 @@
+  Alluxio masters are critical to the cluster and should not be launched as spot instances.
+  In contrast, worker instances fit the use case of spot instances
+  because new workers can register themselves to the cluster and unavailable workers will be marked as lost.
+  Because the addition and loss of workers do not affect basic Alluxio functionality,
+  we can support launching Alluxio clusters with spot instances for workers.
+
+- Alluxio Configuration
+  - **EnableHA** defaults to `No`, which instructs the cluster to launch a single master.
+  If set to `Yes`, three master instances will be provisioned.
+  - **WorkersCount** determines the number of Alluxio workers for the cluster.
+  - **AlluxioRootMountS3BucketName** and **AlluxioRootMountS3BucketPath** are combined form a S3 URI to serve as Alluxio filesystem's root mount.
+  The current user account must have the read/write/list permissions to this S3 URI and the URI will be mounted to the root of the Alluxio file system.
+  - **OpenS3Access** should be set to `Yes` if Alluxio or other services in the cluster will need to access S3 buckets,
+  other than the one defined for the root mount.
+  - **AlluxioProperties** can be set to provide additional Alluxio site properties.
+  Alluxio CFT only provides the necessary parameters to create an Alluxio cluster.
+  If you desire to fine-tune Alluxio behavior, specify the desired properties in the format of `KEY1=VALUE1,KEY2=VALUE2`.
+  The specified key-value pairs will be appended to the `alluxio-site.properties` file in all nodes inside the Alluxio cluster.
+
+- Advanced Alluxio Configuration: These optional parameters are only available in the advanced template.
+  - **AlluxioJournalSize** determines the EBS volume size, in GB, where Alluxio's embedded journal will write to.
+  Each Alluxio master mounts a EBS volume dedicated for logging all metadata changes to achieve fault tolerance.
+  The journal size should be proportional to the estimated magnitude of Alluxio metadata operations.
+  - **AlluxioWorkerMemPercent** is the percentage of the instance's total memory to allocate for the Alluxio worker process.
+  By default this is set to 70%.
+  This space is used for caching remote data in order to drastically increase overall I/O throughput.
+  - **AlluxioWorkerSSDSize** determines the SSD volume size, in GB, to attach to each worker instance.
+  An EBS volume is attached for workers to attach remote data if the value is non-zero.
+  A SSD drive is a simple scalable alternative to instance memory to help increase the Alluxio worker's caching capacity.
+  - **AlluxioMetadataBackupDirectory** determines the path to backup Alluxio's journal contents.
+  This path is relative to Alluxio's root mount point.
+  Backup files can be use to relaunch an Alluxio cluster 
+  - **AlluxioMetadataBackupTime** is the time to schedule the daily backup operation.
+  If set to an empty string, the backup operation will be disabled.
```

Comment:
**AlluxioMetadataBackupDirectory** decides whether backup operation is taken instead of the `AlluxioMetadataBackupTime`